File: app/src/main/java/com/fox/training/fragment/LoveFragment.java
package com.fox.training.fragment;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.fox.training.R;
import com.fox.training.data.model.Music;
import com.fox.training.recyclerview.LoveMusicAdapter;

import java.util.ArrayList;

public class LoveFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View mRootView = inflater.inflate(R.layout.fragment_love, container, false);
        RecyclerView mLoveMusicRecyclerView = mRootView.findViewById(R.id.rvLove);
        ArrayList<Music> mListMusic = new ArrayList<>();

        mListMusic.add(new Music(R.drawable.img_peashooter,
                "Peashooter Song", "Peashooter"));
        mListMusic.add(new Music(R.drawable.img_sunflower,
                "SunFlower Song", "SunFlower"));

        LoveMusicAdapter mMusicAdapter = new LoveMusicAdapter(getContext(), mListMusic);
        mLoveMusicRecyclerView.setLayoutManager(new GridLayoutManager(getContext(), 1));
        mLoveMusicRecyclerView.setAdapter(mMusicAdapter);
        return mRootView;
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
    }
}

File: app/src/main/java/com/fox/training/ui/main/fragment/LoveFragment.kt
package com.fox.training.ui.main.fragment

import android.annotation.SuppressLint
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.fragment.app.Fragment
import com.fox.training.data.network.response.Music
import androidx.recyclerview.widget.LinearLayoutManager
import com.fox.training.data.network.response.DataResult
import com.fox.training.data.source.remote.RemoteDataSource
import com.fox.training.databinding.FragmentLoveBinding
import com.fox.training.ui.main.adapter.TopMusicAdapter
import retrofit2.Call
import retrofit2.Callback
import retrofit2.Response

class LoveFragment : Fragment() {

    private var musicList = mutableListOf<Music>()
    private lateinit var binding: FragmentLoveBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentLoveBinding.inflate(inflater, container, false)
        return binding.root
    }

}